src/libs/ascent/runtimes/flow_filters/ascent_runtime_relay_filters.cpp

```diff
@@ -330,13 +330,85 @@ verify_io_params(const conduit::Node &params,
         }
     }
 
+    if( params.has_child("hdf5_options") )
+    {
+        //
+        // HDF5 OPTIONS Example:
+        //
+        // compact_storage:
+        //   enabled: "true"
+        //   threshold: 1024
+        // chunking:
+        //   enabled: "true"
+        //   threshold: 2000000
+        //   chunk_size: 1000000
+        //   compression:
+        //     method: "gzip"
+        //     level: 5
+        
+        
+        const Node &params_hdf5_opts = params["hdf5_options"];
+
+        res &= check_object("compact_storage",
+                            params_hdf5_opts,
+                            info,
+                            false);
+
+        res &= check_bool("compact_storage/enabled",
+                            params_hdf5_opts,
+                            info,
+                            false);
+
+        res &= check_numeric("compact_storage/threshold",
+                             params_hdf5_opts,
+                             info,
+                             false);
+
+        res &= check_object("chunking",
+                            params_hdf5_opts,
+                            info,
+                            false);
+
+        res &= check_bool("chunking/enabled",
+                          params_hdf5_opts,
+                          info,
+                          false);
+
+
+        res &= check_numeric("chunking/threshold",
+                             params_hdf5_opts,
+                             info,
+                             false);
+
+        res &= check_numeric("chunking/chunk_size",
+                             params_hdf5_opts,
+                             info,
+                             false);
+
+        res &= check_object("chunking/compression",
+                            params_hdf5_opts,
+                            info,
+                            false);
+
+        res &= check_string("chunking/compression/method",
+                            params_hdf5_opts,
+                            info,
+                            false);
+
+        res &= check_numeric("compact_storage/level",
```

Comment:
Based on the example above and test case: 
compact_storage/level -> chunking/compression/level

Reply:
great catch, thank you!